Zend/tests/use_statement/aliasing_builtin_types.phpt

```diff
@@ -25,54 +25,33 @@ $types = [
     "list",
 ];
 
+function run($code) {
+    $php = getenv('TEST_PHP_EXECUTABLE');
+    echo trim(shell_exec("$php -r \"$code\"")), "\n";
```

Comment:
Please only check a single representative error instead, we don't need to enumerate all cases here.